roadmap/3_system_utils_and_dapps/in-progress/chain-key-bitcoin.md

```diff
@@ -8,8 +8,9 @@ is_community: false
 in_beta: true
 ---
 
-Canister smart contracts on the Internet Computer can control real Bitcoin. 
+Canister smart contracts on the Internet Computer can control real bitcoin on the Bitcoin network. 
 However, Bitcoin transactions are slow and expensive. 
-To address that limitation, we aim to introduce a new token called "chain-key Bitcoin", or ckBTC, which is an advanced form of wrapped bitcoin. 
-A canister smart contract that builds on the Bitcoin integration will be able to receive bitcoin and issue ckBTC to the sender, and vice versa, users can use this canister to redeem their ckBTC for real bitcoin.
+To address this limitation, we introduce a new token called "Chain-Key Bitcoin", or ckBTC, which is an analogue of Bitcoin on the Internet Computer. The ckBTC token does not involve traditional wrapping, but builds on the ICP's Bitcoin integration and its chain-key ECDSA signatures.
+A canister smart contract that builds on the Bitcoin integration, including chain-key ECDSA, will be able to receive bitcoin and issue ckBTC to the sender.
```

Comment:
@dietersommer there is some repetition between these two sentences, could you maybe double check that's what you intended?

Reply:
Removed the redundancy.